vm/src/eval.rs

```diff
@@ -22,7 +22,9 @@ mod tests {
     #[test]
     fn test_print_42() {
         let source = String::from("print('Hello world')");
-        let vm = VirtualMachine::default();
+        let mut uninitialized_vm = VirtualMachine::default();
+        let vm = uninitialized_vm.initialize(true);
```

Comment:
Instead of this construction, I would rather see that the `default` function will initialize the `VirtualMachine`. If you want a `VirtualMachine` without initialization, create a new function `VirtualMachine::without_external` which will give the virtual machine without a the external initialization.

Reply:
Thanks for the review.  The default will initialize the vm as normal, and the vm in wasm will be initialized by putting the flag into settings.

Reply:
I takes the suggestion and put the initialize parameters in the settings, such that the vm can also be created and initialized without external.